src/main/scala/regmapper/RegField.scala

```diff
@@ -147,16 +147,16 @@ object RegField
     val numFullBytes = reg.getWidth/8
     val numPartialBytes  = if ((reg.getWidth % 8) > 0) 1 else 0
     val numPadBytes = numBytes - numFullBytes - numPartialBytes
-    val pad = reg | UInt(0, width = 8*numBytes)
-    val oldBytes = Vec.tabulate(numBytes) { i => pad(8*(i+1)-1, 8*i) }
-    val newBytes = Wire(init = oldBytes)
-    val valids = Wire(init = Vec.fill(numBytes) { Bool(false) })
+    val pad = reg | 0.U( (8*numBytes).W)
```

Comment:
```suggestion
    val pad = reg | 0.U((8*numBytes).W)
```

Reply:
Thanks for reviewing! I addressed these changes with the new commit